.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
 <!--
 
-Make sure to run `msbuild /t:pack /v:m` in the repository root when
-you have any of the following changes that affect auto-generated files. Otherwise, the CI build will fail.
+Make sure you have read the contribution guidelines: 
+- https://docs.microsoft.com/contribute/dotnet/dotnet-contribute-code-analysis#contribute-docs-for-caxxxx-rules
+- https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md
+
+If your Pull Request is doing one of the following:
 
 - Adding a new diagnostic analyzer or a code fix
 - Adding or updating resource strings used by analyzers and code fixes
 - Updating analyzer package versions in [Versions.props](../eng/Versions.props)
 
-(Consider merging master into your branch before you run msbuild pack to reduce having merge conflicts)
-
-If you're adding a new rule, Make sure to read the guidlines in: https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md.
-Also, see https://docs.microsoft.com/contribute/dotnet/dotnet-contribute-code-analysis#contribute-docs-for-caxxxx-rules for documentation guidelines.
+Then, make sure to run `msbuild /t:pack /v:m` in the repository root; otherwise, the CI build will fail.
 
+Note: Consider merging master into your branch before you run the pack command to reduce merge conflicts.
```

Comment:
```suggestion
Note: Consider merging the PR base branch (`2.9.x`, `master`, or `release/*` 6) into your branch before you run the pack command to reduce merge conflicts.
```

Reply:
Out of scope for this PR, but there should be instructions for when to target each branch. These instructions should be clear about:

- Do new rules go to `release/6.0.1xx-preview1` or `release/6.0.1xx-preview2`?
- Which fixes to existing rules should go to `2.9.x` and which should go to `master?